ml-agents/mlagents/trainers/learn.py

```diff
@@ -101,6 +101,14 @@ def _create_parser():
         type=int,
         help="Number of parallel environments to use for training",
     )
+
+    argparser.add_argument(
```

Comment:
Would prefer this to not be a CLI command as it affects re-runability. But not obvious where it would go as it would not be inside either behavior's config. @awjuliani @andrewcoh is it time we re-invent the `trainer_config.yaml`?

Reply:
I'm actually considering making this a per trainer parameter so that they can be different for different teams.  Does it make sense that for some games we might want to train one agent team longer than another? An example I can think of is a team of different agents vs a boss. Training the boss might be super simple since it's just one agent but training the team might require learning tons of complex dynamics.  It might make sense to train the team for 1M timesteps for every 100K steps the boss is trained.